test/e2e/ig_latency/ig_latency.csv

```diff
@@ -0,0 +1,23 @@
+method,no-import-group,import-group,change-rate
+GetLocationsRankings,2.059969,3.296367,160.020245%
```

Comment:
Some minor formatting nit, maybe keep 2 decimal for second, and show change rate as +60.1% or -20.1% (the actual change percent)

Reply:
done